server/wfprev-api/src/test/java/ca/bc/gov/nrs/wfprev/CodesControllerTest.java

```diff
@@ -40,69 +46,69 @@ class CodesControllerTest {
+        when(codesService.getAllForestAreaCodes()).thenReturn(facModel);
 
-      mockMvc.perform(get("/codes/" + CodeTables.FOREST_AREA_CODE)
-              .contentType(MediaType.APPLICATION_JSON))
-              .andExpect(status().isOk());
+        mockMvc.perform(get("/codes/" + CodeTables.FOREST_AREA_CODE)
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk());
     }
 
     void testGetGeneralScopeCodes() throws Exception {
-      String exampleId1 = UUID.randomUUID().toString();
-      String exampleId2 = UUID.randomUUID().toString();
+        String exampleId1 = UUID.randomUUID().toString();
+        String exampleId2 = UUID.randomUUID().toString();
 
-      GeneralScopeCodeModel fac1 = new GeneralScopeCodeModel();
-      fac1.setGeneralScopeCode(exampleId1);
+        GeneralScopeCodeModel fac1 = new GeneralScopeCodeModel();
+        fac1.setGeneralScopeCode(exampleId1);
 
-      GeneralScopeCodeModel fac2 = new GeneralScopeCodeModel();
-      fac2.setGeneralScopeCode(exampleId2);
+        GeneralScopeCodeModel fac2 = new GeneralScopeCodeModel();
+        fac2.setGeneralScopeCode(exampleId2);
 
-      List<GeneralScopeCodeModel> facList = Arrays.asList(fac1, fac2);
-      CollectionModel<GeneralScopeCodeModel> facModel = CollectionModel.of(facList);
+        List<GeneralScopeCodeModel> facList = Arrays.asList(fac1, fac2);
+        CollectionModel<GeneralScopeCodeModel> facModel = CollectionModel.of(facList);
 
-      when(codesService.getAllGeneralScopeCodes()).thenReturn(facModel);
+        when(codesService.getAllGeneralScopeCodes()).thenReturn(facModel);
 
-      mockMvc.perform(get("/codes/" + CodeTables.GENERAL_SCOPE_CODE)
-              .contentType(MediaType.APPLICATION_JSON))
-              .andExpect(status().isOk());
+        mockMvc.perform(get("/codes/" + CodeTables.GENERAL_SCOPE_CODE)
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk());
     }
 
     void testGetProjectTypeCodes() throws Exception {
-      String exampleId1 = UUID.randomUUID().toString();
-      String exampleId2 = UUID.randomUUID().toString();
+        String exampleId1 = UUID.randomUUID().toString();
+        String exampleId2 = UUID.randomUUID().toString();
 
-      ProjectTypeCodeModel fac1 = new ProjectTypeCodeModel();
-      fac1.setProjectTypeCode(exampleId1);
+        ProjectTypeCodeModel fac1 = new ProjectTypeCodeModel();
+        fac1.setProjectTypeCode(exampleId1);
 
-      ProjectTypeCodeModel fac2 = new ProjectTypeCodeModel();
-      fac2.setProjectTypeCode(exampleId2);
+        ProjectTypeCodeModel fac2 = new ProjectTypeCodeModel();
+        fac2.setProjectTypeCode(exampleId2);
 
-      List<ProjectTypeCodeModel> facList = Arrays.asList(fac1, fac2);
-      CollectionModel<ProjectTypeCodeModel> facModel = CollectionModel.of(facList);
+        List<ProjectTypeCodeModel> facList = Arrays.asList(fac1, fac2);
```

Comment:
We probably want to rename the variables in this file accordingly, looks like fac1, fac2 etc was copy pasted from the testGetForestAreaCodes() case

Reply:
fixed, thank you.